radicle-httpd/src/api/auth.rs

```diff
@@ -11,19 +11,25 @@ impl Serialize for DateTime {
     }
 }
 
+#[derive(Clone)]
 pub enum AuthState {
     Authorized(Session),
-    Unauthorized {
-        public_key: PublicKey,
-        expires_at: DateTime,
-    },
+    Unauthorized(Session),
 }
 
-// We copy the implementation of siwe::Message here to derive Serialization and Debug
-#[derive(Clone, Serialize)]
-#[serde(rename_all = "camelCase")]
+#[derive(Clone)]
 pub struct Session {
+    pub status: String,
```

Comment:
Not a blocker on my end, but I'm a bit confused why `status` is carried within the `Session`. Isn't that encoded via `AuthState`?

Reply:
Yeah, that's the reason I wanted the serializer add it to the json ouput as a `derive tag`operation.

Will see if I find a better way to handle it.

Reply:
Ok I removed the `AuthState` enum wrapper around `Session`, which seemed to make less sense without SIWE.
Seems nicer altogether
03dae8a